src/components/Parent.vue

```diff
@@ -58,7 +72,15 @@ export default {
       }
     },
 
-    fetchColumns() {},
+    fetchColumns() {
+     const keysArray = (Object.keys(this.myJson[0]));
```

Comment:
this works, but what if `myJson[1]` has different values than `myJson[0]`. An improvement would be to map through all keys in `myJson` and then uniquify the array. Here's how to uniquify easily
https://dev.to/clairecodes/how-to-create-an-array-of-unique-values-in-javascript-using-sets-5dg6